src/System.Reflection.Metadata/tests/PortableExecutable/PEBuilderTests.cs

```diff
@@ -320,8 +347,9 @@ private static MethodDefinitionHandle BasicValidationEmit(MetadataBuilder metada
             return mainMethodDef;
         }
 
-        [Fact]
-        public void Complex()
+        [Theory] // Do BasicValidation on common machine types
+        [MemberData(nameof(CommonMachineTypes))]
```

Comment:
Why not simply

```
[Theory]
[InlineData(new[] { Machine.Unknown, ..., Machine.ArmThumb2 })]
```
?

Reply:
I've just updated this to run for all machine types as well. It doesn't appear to be meaningfully slow.